integration/simulation/l2_network.go

```diff
@@ -13,10 +14,12 @@ import (
 
 // L2NetworkCfg - models a full network including artificial random latencies
 type L2NetworkCfg struct {
-	nodes            []*host.Node
-	enclaveServers   []*grpc.Server
-	avgLatency       uint64
-	avgBlockDuration uint64
+	nodes               []*host.Node
+	nodeTxAddresses     []string
+	nodeRollupAddresses []string
+	enclaveServers      []*grpc.Server
```

Comment:
why do the enclave servers have to be in the L2 network?
they are an implementation detail

Reply:
Removed, thanks.